protostar-rust/src/lib.rs

```diff
@@ -60,7 +56,7 @@ fn find_cairo_files_in_directory(input_path: &Utf8PathBuf) -> Result<Vec<Utf8Pat
 
 fn internal_collect_tests(
     input_path: &Utf8PathBuf,
-    linked_libraries: Option<&Vec<LinkedLibrary>>,
+    linked_libraries: Option<Vec<LinkedLibrary>>,
```

Comment:
Ok, I changed it because otherwise we'd end up with errors from the compiler, I'm not going to spend more time on this, we can clone some arrays of length < 10 2 or 3 times and no one will get hurt and nothing's gonna break. If that's so much of a big deal, you can always submit an issue and we can address this later, it's a micro-optimization that doesn't matter much.